### DISCUSSION TITLE
EntireSpectrum Optical Photon Count [Run issue]:

### DISCUSSION THREAD

**Discussion by VictoriaJPR** [OpenTOPAS/OpenTOPAS#54] [Run-Problem] [answered]
### Contact Details

vjp12@duke.edu

### What was the issue?

I am running a simulation and scoring optical photons (Cherenkov) radiation generation of a drug / material that I created. However, upon scoring the optical photons there is a HARD cutoff at the 400 and 700 nm range. This is the visual light range but I want the ENTIRE range of chereknov which actually peaks at around 300-350 nm. Can someone help me out for scoring this?

### TOPAS category

Scoring

### Version

3.7

### What Operative System (OS) are you running on?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment]
A parameter file would be useful to provide help.
Best!

**Comment by JoseRamosMendez** [comment, marked as answer]
Also, note that TOPAS is now OpenTOPAS v4.0. Topas version 3.7 is deprecated. The optical photon properties changed see [here](https://opentopas.readthedocs.io/en/latest/parameters/physics/optical.html#optical-photons)
